packages/@react-aria/selection/src/useSelectableCollection.ts

```diff
@@ -123,7 +123,10 @@ export function useSelectableCollection(options: AriaSelectableCollectionOptions
   let onKeyDown = (e: KeyboardEvent) => {
     // Prevent option + tab from doing anything since it doesn't move focus to the cells, only buttons/checkboxes
     if (e.altKey && e.key === 'Tab') {
-      e.preventDefault();
+      // allowing default behavior for above and below key
+      if (!delegate.getKeyAbove || !delegate.getKeyBelow) {
+        e.preventDefault();
+      }
```

Comment:
since this is in the `if (e.altKey && e.key === 'Tab') {` statement, this won't actually have the desired effect of allowing default behavior if the the delegate returns null from getKeyAbove/getKeyBelow. You'll want to look at the cases in the switch statement like here: https://github.com/adobe/react-spectrum/blob/449bc07cca60a810c48ebbb8aa3ec242036243ef/packages/%40react-aria/selection/src/useSelectableCollection.ts#L166 and make those `e.preventDefault` not be called if `nextKey` is null I imagine.